src/Plugin.php

```diff
@@ -199,22 +200,29 @@ public function onDependenciesChangedEvent()
      */
     private function loadInstalledPaths()
     {
-        if ($this->isPHPCodeSnifferInstalled() === true) {
-            $this->processExecutor->execute(
-                sprintf(
-                    'phpcs --config-show %s',
-                    self::PHPCS_CONFIG_KEY
-                ),
-                $output,
-                $this->composer->getConfig()->get('bin-dir')
-            );
+        if ($this->isPHPCodeSnifferInstalled() !== true) {
+            return;
```

Comment:
Personally, I am rather opposed to the "early return" pattern. Besides being a way of hiding complexity, it is also a misrepresentation of what is actually happening in the flow.

In this specific instance it also has the negative side-effect of making the changeset a lot larger than it needs to be because of the change in whitespace.

Reply:
Done & done.

Most projects I contribute to have a preference for early return, I'm not that bothered either way.

Anyway, I've changed it around now.